assets/css/wiki.css

```diff
@@ -6,10 +6,6 @@
   font-size: 2em;
 }
 
-.wikisite #editsitecontent svg {
```

Comment:
Not sure about this change. I don't know why I removed this.

Reply:
Ok. It's in fw.css now
https://github.com/jamulussoftware/jamuluswebsite/pull/101/files#diff-4cd73b4b50ea8fd44c12feef9ca19c3d6231f7a052df20996af1fca89648a9caR182